Xamarin.PropertyEditing.Mac/Controls/PredefinedValuesEditor.cs

```diff
@@ -184,7 +184,9 @@ private void PopupButton_Activated (object sender, EventArgs e)
 
 		private void ComboBox_SelectionChanged (object sender, EventArgs e)
 		{
-			if (ViewModel != null && this.comboBox != null) {
+			if (ViewModel != null
+				&& this.comboBox != null
+				&& this.comboBox.SelectedValue != null) {
```

Comment:
&& this.comboBox?.SelectedValue

Reply:
@netonjm It won't read the 3rd check if `this.combobox` is null. It will bail straight away because 1 of the 1st 2 checks has failed. The compiler usually checks the conditionals in sequence.